changelogs/3.1.0.md

```diff
@@ -0,0 +1,253 @@
+# 3.1.0
+
+Date: 2024-04-15
+Tag: 3.1.0
+
+## Overview
+
+
+3.1.0 is the 1st [stable][release_policy] version of the 3.1 release
+series. It introduces 31 improvements and resolves 35 bugs since
+3.0.1.
+
+The "stable" label means that we have all planned features implemented and we
+see no high-impact issues. However, if you encounter an issue, feel free to
+[report it][issues] on GitHub.
+
+[release_policy]: https://www.tarantool.io/en/doc/latest/dev_guide/release_management/#release-policy
+[issues]: https://github.com/tarantool/tarantool/issues
+
+
+## Compatibility
+
+Tarantool 3.1.x is backward compatible with Tarantool 2.11.x in the binary data
```

Comment:
Just wondering, why mention 2.11 here? It's not the latest previous version, and nothing is said about other versions. What about 3.0, 2.10, 2.8.4 which are [still supported](https://www.tarantool.io/en/doc/latest/release/)?

Reply:
We always refer to the lates LTS. Before it was 1.10.x, now we moved to 2.11.x.